ee/vercel/integration.py

```diff
@@ -216,46 +216,29 @@ def _cleanup_failed_installation(
 
     @staticmethod
     def get_vercel_plans() -> list[dict[str, Any]]:
-        # TODO: Retrieve through billing service instead.
+        # Single usage-based plan matching posthog.com model
+        # Billing is handled through PostHog's billing service, not Vercel
         return [
             {
-                "id": "free",
+                "id": "posthog-usage-based",
                 "type": "subscription",
-                "name": "Free",
-                "description": "No credit card required",
+                "name": "PostHog",
+                "description": "Usage-based analytics. First 1M events free. View pricing: https://posthog.com/pricing",
                 "scope": "installation",
                 "paymentMethodRequired": False,
```

Comment:
Shouldn't it be required? Otherwise it seems like they could subscribe to this plan without payment method added on Vercel?

Reply:
Can we have customers signing up to PostHog if they don't have a CC on Vercel? In a way I feel like the integration would be more robust if we always required "serious" customers - those who have a CC on file - but on the other hand we dont want it to be hard for new users.

What do you mean by "keeping as is on master"? What are the different options here?